.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help improve Quiltflower
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Quiltflower version**
+Replace this text with the version of Quiltflower that you are using.
+
+**Describe the bug**
```

Comment:
I think the bold text should be a header instead

Reply:
I agree. I'll make this change.